src/loggingFunctions.js

```diff
@@ -9,14 +9,14 @@ Object.keys(config._logLevels).forEach(function(logLevel) {
 		logFunction: logLevel === 'trace' ? logUtil.logTraceWithColor : logUtil.logWithColor,
 		level: config._logLevels[logLevel].level,
 		functionName: config._logLevels[logLevel].letter,
-		color: config.colors[logLevel],
+		color: config._types.log === 0 ? config.colors.logs[logLevel] : config.colors.lines[logLevel],

```

Comment:
lol. It actually took me an hour to find out how this line works and how explicit the error was.

`config._types.log === 0` is always true. 😆 😆